packages/common/src/intl/IntlTestProvider.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from 'react';
+import { IntlProvider as ReactIntlProvider } from 'react-intl';
+import en from './locales/en.json';
+
+export const IntlTestProvider: FC = ({ children }) => {
+  return (
+    <ReactIntlProvider locale="en" messages={en}>
+      {children}
+    </ReactIntlProvider>
+  );
```

Comment:
The testing-library doesn't like it when you change state internally 🤔  (The real intl provider does this to lazy load the translations).

it gives you functionality to, for example, press a button or input text etc. The preference is that these state changing invocations are done within a method `act` - which ensures that the component is fully rendered and everything is all good before trying to do state changes.

So, if you change internal state outside of an `act`, the library gives a warning. I thought that this component was OK and that because we require quite a few different providers for components, it might be best to make a `TestProvider` which can actually pass a lot more things initially - we could then pass through props the `messages` to test different localisations etc.